lib/components/PathProgress/PathProgress.tsx

```diff
@@ -54,24 +55,19 @@ export const PathProgress = ({ steps = 5, currentStep = 0, selectedStep, onClick
           `)}
           onClick={handleOnClick(index + 1)}
         >
-          {selectedStep === index + 1 ? (
-            <span className="material-symbols-outlined ds-font-bold">check</span>
-          ) : (
-            index + 1
-          )}
+          {selectedStep === index + 1 ? <MdCheck size={24} /> : index + 1}
```

Comment:
In the design the check icon is little heavier. Tried to use `strokeWidth` prop for the icon, but it does introduce weird outline for the icon also.
Is it okay to be like this, a little thinner version or what do other think about?

Reply:
I checked it out, and it looks fine to me